services/jenkins/jenkins-coverage.service.js

```diff
@@ -47,9 +48,48 @@ const formatMap = {
       )
       return { coverage: lineCoverage.ratio }
     },
+    pluginSpecificPath: 'cobertura',
+  },
+  api: {
+    schema: Joi.object({
+      results: Joi.object({
+        elements: Joi.array()
+          .items(
+            Joi.object({
+              name: Joi.string().required(),
+              ratio: Joi.number()
+                .min(0)
+                .max(100)
+                .required(),
+            })
+          )
+          .has(Joi.object({ name: 'Line' }))
+          .min(1)
+          .required(),
+      }).required(),
+    }).required(),
+    treeQueryParam: 'results[elements[name,ratio]]',
+    transform: json => {
+      const lineCoverage = json.results.elements.find(
+        element => element.name === 'Line'
+      )
+      return { coverage: lineCoverage.ratio }
+    },
+    pluginSpecificPath: 'coverage/result',
   },
 }
 
+const documentation = `
+<p>
+  We support coverage metrics from a variety of Jenkins plugins:
+  <ul>
+    <li><a href="https://plugins.jenkins.io/jacoco">JaCoCo</a></li>
+    <li><a href="https://plugins.jenkins.io/cobertura">Cobertura</a></li>
+    <li>Any plugin which integrates with the <a href="https://plugins.jenkins.io/code-coverage-api">Code Coverage API</a> (e.g. llvm-cov, Cobertura 1.13+, etc.)</li>
+  </ul>
+</p>
+`
```

Comment:
👍

Reply:
Actually I just noticed some weird formatting on the list items in the review app:
![image](https://user-images.githubusercontent.com/13042488/66704388-2390ce00-ece1-11e9-8ed1-9085f0e74592.png)

Reply:
Hmmmm, interesting, I've got a different result: ![jenkins-coverage](https://user-images.githubusercontent.com/10694593/66704506-64560a80-ed14-11e9-9d20-458ae927c914.png)

Do you have strange results with other badges that have lists in their documentation text (e.g. Matrix)?

Reply:
Yeah I am seeing something similar, although it is slightly less pronounced since the individual list items are closer in length. Not a blocker for this by any means, but perhaps something we want to dig into later on. 

I'm seeing this in Chrome on Win 10 FWIW

Reply:
I managed to reproduce this in Opera so it's probably a Chromium vs. Firefox thing. Will open a separate issue.

Reply:
See #4170.